proc/proc_test.go

```diff
@@ -2374,8 +2373,25 @@ func TestIssue683(t *testing.T) {
 				break
 			}
 		}
-		if !goterror {
-			t.Fatal("expeceted an error we didn't get")
+		ver, _ := ParseVersionString(runtime.Version())
+
+		if !(ver.Major < 0 || ver.AfterOrEqual(GoVersion{1, 8, -1, 0, 0})) {
+			// in go1.8 the debug_line entry is correct for this piece of code
+			if !goterror {
```

Comment:
I think it's better if I remove the error check completely, whether or not we get the error is irrelevant and depends on what the compiler does, what's important is that the test doesn't panic.